include/CFinalBoneHierarchy.h

```diff
@@ -100,6 +101,35 @@ namespace scene
                 createAnimationKeys(inLevelFixedJoints);
             }
 
+			void fillExportBlob(void* _dataPtr, uint32_t* _size)
+			{
+				new (_dataPtr) core::FinalBoneHierarchyBlobV1(boneCount, NumLevelsInHierarchy, keyframeCount);
+				uint8_t* ptr = (uint8_t*)_dataPtr + sizeof(core::FinalBoneHierarchyBlobV1);
```

Comment:
On another note, maybe instead of doing the copying here... do it in the constructor of the FinalBoneHierarchyBlobV1 class?
At least for consistency :)

We can get all the private members of CFinalBoneHierarchy, i.e.
keyframeCount == CFinalBoneHierarchy::getKeyframeCount()
keyframes == CFinalBoneHierarchy::getKeys()
interpolatedAnimations == CFinalBoneHierarchy::getInterpolatedAnimationData(0)
nonInterpolatedAnimations == CFinalBoneHierarchy::getNonInterpolatedAnimationData(0)
etc.